.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -14,6 +14,12 @@ If you're not sure what version you're using, run `print scalaVersion` from sbt
 
 ## Minimized code
 
+<!--
+This code should be self contained, compilable (with possible failures) and as small as possible.
+
+Ideally, we should be able to just copy this code in a file and run `scalac` (and maybe `scala`) to reproduce the issue.
```

Comment:
How about also asking for providing the exact command used to run the code? Some bugs are reproducible only with specific compiler flags.
Alternatively maybe we could encourage people to provide reproductions with scala-cli? From my experience (and as I'm dealing with the responsibilities of the issue tsar now) this is really convenient. It allows you to specify all compiler options, scala version java version and others in a single file together with your code. Then we could potentially even automate checking reproducibility of reported issues

Reply:
The immediate goal to avoid issues with partial code snippets that cannot be reasoned about. We can always improve it later and add a complete guide. 

Not sure if `scala-cli` would be that helpful right now. What need is tests that can easily be placed in the tests of the compiler. At some point we could enhance our testing framework to understand the `scala-cli` directives but it seems a bit early.

Reply:
I'm quite convinced it would be helpful but of course this is not something we can't live without right now